sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, Post, Param } from "@nestjs/common";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty } from "../../auth/decorators";
+import { CreateEducationProgramOfferingDto } from "./models/create-education-program-offering.dto";
+import { EducationProgramOfferingService } from "../../services";
+
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@Controller("institution/offering")
+export class EducationProgramOfferingController {
+  constructor(
+    private readonly programOfferingService: EducationProgramOfferingService,
+  ) {}
+
```

Comment:
Please add the authorization attribute for the location id.
```
@HasLocationAccess("locationId")
```

Reply:
done